docs/WhiteRabbit.md

```diff
@@ -193,6 +194,21 @@ However these two values (1 & 2) are not the only values present in the data bec
 These other values appear with very low frequency (defined by “Min cell count”) and often represent incorrect or highly suspicious values.
 When generating an ETL we should not only plan to handle the high-frequency gender concepts 1 and 2 but the other low-frequency values that exist within this column.
 
+#### Numerical statistics (develop)
+If the option for numerical statistics is checked, then a set of statistics is calculated for all integer, real and date data types. 
+The following statistics are added to the Overview sheet:
+
+* Average
+* Standard Deviation (sample)
+* Minimum
+* Maximum
+* Quartiles (sample)
+
+Depending on the set reservoir size, some statistics are approximations rather than true population statistics.
+If the number of values is smaller than the reservoir size, then the standard deviation and three quartile boundaries are based on only the sample.
```

Comment:
> If the number of values is smaller than the reservoir size, then the standard deviation and three quartile boundaries are the exact statistics. Otherwise, the statistics are based on a representative sample.